lib/schemaUtils.js

```diff
@@ -1686,6 +1683,14 @@ module.exports = {
       enumValue,
       formHeaders = [];
 
+    let concreteUtils = DEFAULT_SCHEMA_UTILS;
```

Comment:
why do we need to define the concrete schema utils handler if this was defined in schemapack? also if this code is necessary it is repeated in safeSchemaFaker function, should be moved to an independent function and reutilize it